shell/platform/darwin/ios/framework/Source/platform_views_controller.mm

```diff
@@ -194,114 +346,113 @@ bool ClipRRectContainsPlatformViewBoundingRect(const SkRRect& clip_rrect,
     }
   }
 
-  NSObject<FlutterPlatformView>* embedded_view = [factory createWithFrame:CGRectZero
-                                                           viewIdentifier:viewId
-                                                                arguments:params];
-  UIView* platform_view = [embedded_view view];
+  NSObject<FlutterPlatformView>* embeddedView = [factory createWithFrame:CGRectZero
+                                                          viewIdentifier:viewId
+                                                               arguments:params];
+  UIView* platformView = [embeddedView view];
   // Set a unique view identifier, so the platform view can be identified in unit tests.
-  platform_view.accessibilityIdentifier =
-      [NSString stringWithFormat:@"platform_view[%lld]", viewId];
+  platformView.accessibilityIdentifier = [NSString stringWithFormat:@"platform_view[%lld]", viewId];
 
-  FlutterTouchInterceptingView* touch_interceptor = [[FlutterTouchInterceptingView alloc]
-                  initWithEmbeddedView:platform_view
-               platformViewsController:GetWeakPtr()
-      gestureRecognizersBlockingPolicy:gesture_recognizers_blocking_policies_[viewType]];
+  FlutterTouchInterceptingView* touchInterceptor = [[FlutterTouchInterceptingView alloc]
+                  initWithEmbeddedView:platformView
+               platformViewsController:self
```

Comment:
is changing to strong intended?

Reply:
Yes. This is passing a strong `self` var, but the receiving side is holding it in a weak ivar. No need to declare a `weakSelf` since we're not capturing it in a block, etc.

That said, If I haven't declared the parameter as a `__weak` param that would probably be useful, just to make it more obvious from the API that this is intended to be weakly held. I'll do that.